### DISCUSSION TITLE
24 - Custom Theming

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#1298] [RFCs/Ideas]
This is to discuss Dendron's [custom theming](https://wiki.dendron.so/notes/PjXL1NKL7yHV0ExHnROBl.html#goals) proposal

**Comment by ghost** [comment]
I think you should change the link to the following to have it link to the correct RFC: https://docs.dendron.so/notes/PjXL1NKL7yHV0ExHnROBl/#goals

**Comment by ScriptAutomate** [comment]
Related: https://github.com/dendronhq/dendron/issues/496
